doc/designs/ddd/WINDOWS.md

```diff
@@ -2,11 +2,11 @@ Windows-related issues
 ======================
 
 Supporting Windows introduces some complications due to some "fun" peculiarities
-of Windows's socket API.
+of Windows socket API.
```

Comment:
> The possessive form of "Windows" has been updated from "Windows's" to "Windows'".

One too many---the apostrophe disappeared :D

Reply:
IMO OK without the apostrophe.

Reply:
I also find `Windows socket API` is already very understandable. Just like we usually use the term `Linux/Unix socket programming` instead of `Linux's socket...`

Reply:
Should be `Windows'` - apostrophe at the end. It's hard to see in @bbbrumley's comment

Reply:
It should either be `... peculiarities of the Windows socket API` or `... peculiarities of Windows' socket API`. I think the former would be better

Reply:
Ping @ha1215 `peculiarities of Windows socket API` isn't right, and should preferably be changed to  `peculiarities of the Windows socket API` (avoids all the apostrophe-after-s concerns)

Reply:
This is still wrong IMO